src/cryptography/hazmat/decrepit/ciphers/algorithms.py

```diff
@@ -90,3 +90,18 @@ def __init__(self, key: bytes):
     @property
     def key_size(self) -> int:
         return len(self.key) * 8
+
+
+# This class only allows RC2 with a 128-bit key. No support for
+# effective key bits or other key sizes is provided.
+class RC2(BlockCipherAlgorithm):
```

Comment:
Should it be named `RC2_128` or something? Or is the idea that if we ever _did_ want to support more key sizes, we'd just add more args to the constructor?

Reply:
Yeah it's actually not very hard to support the effective key bits stuff so I'd rather just have the "generic" class. I had support and ripped it out since I want to see someone come with a use case.